question/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Answer(models.Model):
+  """Answer model."""
+
+  is_correct = models.BooleanField(default=False)
+  question = models.ForeignKey('Question', on_delete=models.CASCADE)
```

Comment:
Consider refactoring: replace Answer FK with Question M2Ms (one for the correct and another for the incorrect answers)